examples/operator/operator.yaml

```diff
@@ -403,6 +403,30 @@ spec:
                               information specific to the underlying infrastructure
                               provider.
                             properties:
+                              aws:
```

Comment:
__Note__: This is leftover from https://github.com/projectcontour/contour-operator/pull/336